src/ucp/core/ucp_ep.h

```diff
@@ -610,6 +610,9 @@ ucs_status_t ucp_worker_create_mem_type_endpoints(ucp_worker_h worker);
 
 void ucp_worker_destroy_mem_type_endpoints(ucp_worker_h worker);
 
+void ucp_worker_mem_type_endpoints_print_info(ucp_worker_h worker,
```

Comment:
may be ucp_worker_mem_type_eps_print_info?
to unify with other function names?

Reply:
it is unified with `mem_type`-related functions which use `*mem_type_endpoints*`

Reply:
maybe it is worth changing their names too?